docker/docker-python/Dockerfile.mxnet.python3.cpu

```diff
@@ -0,0 +1,28 @@
+# -*- mode: dockerfile -*-
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Dockerfile to build MXNet for CPU
+
+FROM ubuntu:16.04
+
+RUN apt-get update
+RUN apt-get install -y wget python3 gcc
+RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN python3 get-pip.py
```

Comment:
Looks good.  Super nit: I'd combine smaller commands like this into a single layer.  
```Dockerfile
RUN apt-get update && \
        apt-get install -y wget python3 gcc && \
        ...
```

Reply:
@KellenSunderland @bhavinthaker Thank you, I agree. I will update the PR.